### PR TITLE
Expand README files

### PR DESCRIPTION
**Pull request by abrown**
This change includes:
 - tweaks to the README formatting
 - duplication of the licensing information (since this will be displayed on crates.io, npmjs.com)
 - clarifications on building and running the various components
 - tweaks to wording

### CODE REVIEW

File: assemblyscript/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "wasi-nn",
```
Comment (abrown) [inline assemblyscript/package.json:2]: @brianjjones, just a thought: should we rename the NPM version of this to `as-wasi-nn`? That would 1) match the `as-wasi` naming, and 2) make it more clear that this is an AssemblyScript package, not a JS or TS one.

Comment (brianjjones) [reply assemblyscript/package.json:2]: I have no issue with that. Should the rust version be rust-wasi-nn then?

Comment (abrown) [reply assemblyscript/package.json:2]: Everything on crates.io is pretty much Rust so I don't think so.

File: README.md [hunk 2]
```diff
@@ -15,35 +13,65 @@
...
+[AssemblyScript README]: assemblyscript/README.md
+
+
+### Examples
+
+This repository includes examples of using these bindings. See the [Rust example] and
+[AssemblyScript example] to walk through an end-to-end image classification using an AlexNet model.
+Run them with:
+ - `./build.sh rust` runs the [Rust example]
+ - `./build.sh assemblyscript` runs the [AssemblyScript example]
```
Comment (brianjjones) [inline README.md:49]: Should be ./build.sh as for the AssemblyScript example.

**Review by brianjjones — APPROVED** [review]

**Pull request merged**